sw/device/lib/usbdev.c

```diff
@@ -251,7 +251,8 @@ void usbdev_endpoint_setup(usbdev_ctx_t *ctx, int ep, int enableout,
   }
 }
 
-void usbdev_init(usbdev_ctx_t *ctx) {
+void usbdev_init(usbdev_ctx_t *ctx, bool pinflip, bool diff_rx, bool diff_tx) {
+  int phy_config;
```

Comment:
Don't predeclare variables; move this down to where it's used, intead.

Also, the type of this value aught to be `uint32_t`.

Reply:
Done